chain-connect/src/customClients/TrustConnectClient.ts

```diff
@@ -91,72 +89,14 @@ function getTrustWalletFromWindow() {
   return window["trustwallet"] ?? null;
 }
 
-export class GalachainConnectTrustClient extends WebSigner {
+export class GalachainConnectTrustClient extends BrowserConnectClient {
```

Comment:
For naming consistency, both the file and class name here should be "TrustWalletConnectClient"